server/src/stt/coqui-stt/parser.js

```diff
@@ -0,0 +1,98 @@
+import wav from 'node-wav'
+import fs from 'fs'
+
+import log from '@/helpers/log'
+
+log.title('Coqui-ai Parser')
+
+const parser = { }
+let STT = { }
+
+/* istanbul ignore next */
+try {
+  STT = require('stt-gpu') // eslint-disable-line global-require, import/no-unresolved
```

Comment:
Coqui only releases TFLite models, so there's no GPU needed. `stt-gpu` is depreciated. You can read more detail [here](https://github.com/coqui-ai/STT/discussions/1922).